common/source/docs/common-prearm-safety-checks.rst

```diff
@@ -253,9 +262,45 @@ Pre-arm checks that are failing will also be sent as messages to the GCS while d
     Throttle below failsafe                                 RC throttle input is below FS_THR_VALUE             Turn on RC transmitter or check :ref:`FS_THR_VALUE<FS_THR_VALUE>`.  Check :ref:`RC failsafe setup<radio-failsafe>`
     Vehicle too far from EKF origin                         Vehicle is more than 50km from EKF origin           Reboot autopilot to reset EKF origin to current Location
     winch unhealthy                                         Winch is not communicating with autopilot           Check winch's physical connection and :ref:`configuration <common-daiwa-winch>`
-[/site]
     ======================================================= =================================================== ====================================================
+[/site]
+[site wiki="plane"]
+
+.. table:: Pre-Arm Failure Messages (Plane Only)
+    :widths: auto
 
+    ======================================================= =================================================== ====================================================
+    Message                                                 Cause                                               Solution
+    ======================================================= =================================================== ====================================================
+    ADSB threat detected                                    ADSB failsafe.  Manned vehicles nearby              See :ref:`ADSB configuration<common-ads-b-receiver>`
+    AHRS not healthy                                        AHRS/EKF is not yet ready                           Wait.  Reboot autopilotFS_LONG_TIMEOUT < FS_SHORT_TIMEOUT
```

Comment:
```suggestion
    AHRS not healthy                                        AHRS/EKF is not yet ready                           Wait.  Reboot autopilot
```

Reply:
oops....have to fix that....alphabetizing copy paste error